app/plugins/NotFound.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace GanttDashboard\App\Plugins;
+
+use Exception;
+use Phalcon\Events\Event;
+use Phalcon\Mvc\Dispatcher;
+use Phalcon\Mvc\Dispatcher\Exception as DispatchException;
+
+class NotFound
+{
+    public function beforeException(Event $event, Dispatcher $dispatcher, Exception $exception)
+    {
+        /**
+         * Default error controller and action
+         */
+        $action = 'index';
```

Comment:
why is this forwarding sometimes to index and other times to notFound?
isn't there 1 single page that says: page not found?

Reply:
You are right.Refactored.